python/py-catalogue/Portfile

```diff
@@ -38,7 +38,12 @@ if {${name} ne ${subport}} {
     depends_build-append \
                     port:py${python.version}-setuptools
 
-    depends_run-append \
+    if {${python.version} < "38"} {
+        depends_build-append \
+                    port:py${python.version}-importlib-metadata
+    }
```

Comment:
this is imported in the files installed by ```py-catalogue``` (i.e., ```/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/catalogue.py```). So this is for sure a runtime dependency. If it's needed at build-time as well you should add it as ```lib``, otherwise as ```run```.

Reply:
Thanks done. 🦃🦃🦃